### PR TITLE
Add logging, format metric values, add min_epoch #17

### PR DESCRIPTION
**Pull request by malabikasen**

### CODE REVIEW

File: config/config.yaml [hunk 1]
```diff
@@ -56,6 +56,7 @@ input_settings:
               hidden: [2]
               channels: [128]
               epochs: [2000]
+              min_epochs: [10]
```
Comment (tmmurali) [inline config/config.yaml]: Is there a good reason for the default to be such a small number?

**Review by tmmurali — CHANGES_REQUESTED** [review]

File: config/config.yaml (hunk 1 above)
Comment (malabikasen) [reply config/config.yaml]: That's a mistake - should have been `1000` as mentioned in the manuscript.

File: config/config.yaml [hunk 2]
```diff
@@ -56,7 +56,7 @@ input_settings:
               hidden: [2]
               channels: [128]
               epochs: [2000]
-              min_epochs: [10]
+              min_epochs: [1000]
```
Comment (malabikasen) [inline config/config.yaml:59]: @tmmurali set `min_epochs `to `1000`

**Comment by tmmurali** [comment]
Thanks. Please go ahead and merge this PR.

**Pull request merged**